deployment/data-feeds/changeset/deploy.go

```diff
@@ -76,3 +77,34 @@ func DeployAggregatorProxy(chain deployment.Chain, aggregator common.Address, ac
 	}
 	return resp, nil
 }
+
+func DeployBundleAggregatorProxy(chain deployment.Chain, aggregator common.Address, owner common.Address, labels []string) (*types.DeployBundleAggregatorProxyResponse, error) {
+	proxyAddr, tx, proxyContract, err := bundleproxy.DeployBundleAggregatorProxy(chain.DeployerKey, chain.Client, aggregator, owner)
+	if err != nil {
+		return nil, fmt.Errorf("failed to deploy BundleAggregatorProxy: %w", err)
+	}
+
+	_, err = chain.Confirm(tx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to confirm BundleAggregatorProxy: %w", err)
+	}
+
+	// AggregatorProxy contract doesn't implement typeAndVersion interface, so we have to set it manually
```

Comment:
BundleAggregatorProxy actually implements [typeAndVersion](https://github.com/smartcontractkit/chainlink-evm/blob/develop/contracts/src/v0.8/data-feeds/BundleAggregatorProxy.sol#L14), so we should use it.

Reply:
Thanks, updated it.